Assets/_Laz/Scripts/UserInterface/DebugBehaviour/DebugKeyHandler.cs

```diff
@@ -0,0 +1,55 @@
+using UnityEngine.InputSystem;
+
+namespace Laz
+{
+    public partial class DebugUIBehaviour
+    {
+        private int GetDebugBuildIndex()
+        {
+            int buildIndex = -1;
```

Comment:
why not just default at 0? So you don't have to do an if check later on?

Reply:
Since a build Index of 0 is still technically valid. So a number of -1 signals that no valid index was pressed and to not go ahead with scene switching

Reply:
so technically you can press LShift + Q, and this method will still get called?

Reply:
Yeah as long as Lshift is being held down, the method gets called